.gitignore

```diff
@@ -1,6 +1,7 @@
 db/postgres*.jar
 .vscode/
 env/
+venv/
```

Comment:
I think we should put this change in a separate PR if we want to add this, it's outside the scope of the change.